app/views/discovery/_carousel.html.haml

```diff
@@ -3,14 +3,15 @@
   %h3{:class => header_name}
     =local_assigns[:tag]
   - if local_assigns[:type] == "vendor"
-    - if local_assigns[:list]
-      - local_assigns[:list].each do |vendor|
+    - if local_assigns[:list] and local_assigns[:list].length >= 4
```

Comment:
I think we should be pruning all tags with less than 4 items in DiscoveryController's index method or another place like that. We should be trying to avoid conditional logic in the views.

Reply:
Also do logic for no more than 15 items outside the view.

Reply:
Yeah, I think 15 items is a good number

Reply:
I fixed this, but forgot to remove the if statement. It is not needed.